spec/services/volunteers_emails_export_csv_service_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe VolunteersEmailsExportCsvService do
+  subject { described_class.new.perform }
+
+  before do
+    create(:volunteer)
+  end
+
+  it "creates CSV" do
+    results = subject.split("\n")
+    expect(results.count).to eq(2)
+    expect(results[0].split(",")).to eq([
+      "Email",
+      "Case Number"
+    ])
+    expect(results[1].split(",").count).to eq(2)
```

Comment:
This all looks really good! Do you think you could give the volunteer some cases and confirm that the 2nd row data looks as you would expect? Can you also include a volunteer who is not active and add make sure that their data does not show up in this list?

Reply:
Sure! I'll work on it